### ISSUE TITLE
mApplyLogitsPostProcessorBatched parameter for batch_manager::GenericLlmRequest 

### ISSUE THREAD

**Issue by akhoroshev** [NVIDIA/TensorRT-LLM#1801]
`batch_manager::GenericLlmRequest` [has](https://github.com/NVIDIA/TensorRT-LLM/blob/main/cpp/include/tensorrt_llm/batch_manager/llmRequest.h#L65C33-L65C115) logitsPostProcessor with type `std::function<void(RequestIdType, TensorPtr&, BeamTokens const&, TStream const&)>;` and [mApplyLogitsPostProcessorBatched](https://github.com/NVIDIA/TensorRT-LLM/blob/main/cpp/include/tensorrt_llm/batch_manager/llmRequest.h#L79) option

How can this type of callback handle a batch of requests?

**Comment by hijkzzz** [comment]
@MartinMarciniszyn @Shixiaowei02 
Could you please help to comment on this issue?
Thanks.

**Comment by MartinMarciniszyn** [comment]
The batched logits postprocessor has this signature:

```
using LogitsPostProcessorBatched = std::function<void(std::vector<IdType> const&, std::vector<Tensor>&,
    std::vector<std::reference_wrapper<BeamTokens const>> const&, StreamPtr const&)>;
```

See [types.h](https://github.com/NVIDIA/TensorRT-LLM/blob/main/cpp/include/tensorrt_llm/executor/types.h#L58) for details.

**Comment by akhoroshev** [comment]
@MartinMarciniszyn batch_manager does not accept such signature. Your link from Executor API not from batch_manager API.

**Comment by MartinMarciniszyn** [comment]
Setting the batched logits processor is not exposed on `GptManager`. Please use the Executor API for this functionality.
